cypress/support/assistant-dummy-llm.js

```diff
@@ -27,7 +29,11 @@ const server = http.createServer((req, res) => {
       // Why add a delay here? reference: https://github.com/opensearch-project/ml-commons/issues/1894
       setTimeout(() => {
         if (requestBody.includes(MATCH_AGENT_FRAMEWORK_PROMPT)) {
-          return res.end(JSON.stringify(agentFrameworkJson));
+          if (requestBody.includes(TOOL_RESPONSE)) {
```

Comment:
Is this `TOOL RESPONSE` request from the `ml-commons` side? Since we are adding  a specific response for `agentFrameworkThoughtJson`, could we just add a condition just like below. Preferring add `else if(requestBody.includes(MATCH_THOUGHT_PROMPT))) {`.

Reply:
Yes, `TOOL RESPONSE` comes from ml-common side and it's part of prompt when we have tool involved.

The first LLM call prompt is `MATCH_AGENT_FRAMEWORK_PROMPT....`, when some tool been involved, prompt becomes `MATCH_AGENT_FRAMEWORK_PROMPT.... TOOL_RESPONSE:....`, i am not quite follow your comments here.

Reply:
Got your point, Let's keep current implementation